examples/existing_vpc_existing_subnets/main.tf

```diff
@@ -2,6 +2,10 @@ provider "aws" {
   region = "us-west-2"
 }
 
+variable "azs_to_deploy" {
```

Comment:
Let's rename this to `max_availability_zones` as it's more clear that it's a limit.